core/include/detray/definitions/indexing.hpp

```diff
@@ -258,4 +286,13 @@ DETRAY_HOST_DEVICE constexpr decltype(auto) get(
 
 }  // namespace detail
 
+/// Overload to check for an invalid typed index link @param ti
+template <typename id_t, typename index_t, typename value_t, value_t id_mask,
```

Comment:
Since the typed index encodes type and index on the same underlying value it needs its own implementation of when it is invalid, just like the barcode. This assures that a call on the ```is_invalid_value``` util does the right thing